### DISCUSSION TITLE
U Got that Look!, Anyone handy with HTML / CSS?

### DISCUSSION THREAD

**Discussion by ldijkman** [ldijkman/randomnerd_esp32_wifi_manager#5] [Ideas]
U Got that Look!

Electra looks a bit HTMLisch

Maybe a darker Skin, more app alike styled would be better for Electra

Anyone handy with HTML / CSS?

Help Electra?

https://github.com/ldijkman/randomnerd_esp32_wifi_manager/tree/main/Skins

**Comment by sergiocntr** [comment]
Hi man!
Great job,I've done a similar project [here](https://github.com/sergiocntr/cadolab100) dark skin ,maybe you could consider it.
Ciao 
Sergio
![Screenshot_20211224-225518](https://user-images.githubusercontent.com/7195587/153771055-898862c4-4dea-4933-8f21-edbcdc79a856.jpeg) 
![Screenshot_20220111-085652](https://user-images.githubusercontent.com/7195587/153771083-39a753df-9f02-44fc-9f75-465e88e06726.jpeg)

**Comment by ldijkman** [comment]
Wow, you are a proffessional designer
Looks super
Super neat your pages

Greet luberth
SkinManager ;-) https://youtu.be/1gPBMtsRDFA

https://youtube.com/user/LuberthDijkman



-
